templates/docker-compose.yml.j2

```diff
@@ -65,6 +65,10 @@ services:
       --websocket-secure-cert-path={{ nim_waku_websocket_ssl_cert | mandatory }}
 {% endif %}
 {% endif %}
+{% if nim_waku_dns_disc_enabled %}
+      --dns-discovery=true
+      --dns-discovery-url={{ nim_waku_dns_disc_url }}
```

Comment:
You'll need to add `| mandatory` to avoid setting it to mothing.

Reply:
Added in https://github.com/status-im/infra-role-nim-waku/pull/4/commits/fdaddbae2144de8faaafe8d2366f56e0b64db31c